### ISSUE TITLE
[Feature Request] Add/remove channel to profile from channel/video page

### ISSUE THREAD

**Issue by ThatIsAPseudo** [FreeTubeApp/FreeTube#568]
**Is your feature request related to a usage problem (not a bug)? Please describe.**
When I subscribe to a channel, I often want to add it to a profile immediately, so it's quite annoying you must go into the profile setting, look for the channel in the list, to finally add it to **one** profile.

**Describe the solution you'd like to see implemented**
There should be a dropdown list next to the subscribe button, to select in which profile(s) this channel is included.

**Comment by GilgusMaximus** [comment]
Yep this is a good idea. Definitely worth to implement. However some new overlaying UI might be required for that, depending on the implementation. 
So this is probably not gonna make it from our side immediately into the next release. 

However if someone else want's to implement that, we are open for pull requests and can give you advise if needed.

**Comment by PrestonN** [comment]
This isn't quite the solution you're looking for, however you can switch your active profile while you're on the channel / video page and subscribing again will add that channel to the new profile.

I do agree that this is something that should be added though.
